MathsOperations/src/test/java/com/mavenproject/first/MathsOperationsTest.java

```diff
@@ -0,0 +1,42 @@
+package com.mavenproject.first;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class MathsOperationsTest {
+
+    MathsOperations mathsOperations;
+
+    @Before
+    public void init() {
+         mathsOperations = new MathsOperations();
+    }
+
+    @Test
+    public void additionTest() {
```

Comment:
since you have already written @Test you can exclude Test in method name. i.e additionTest couls be just addition()

Reply:
okay sir. I will work on above point.